mlflow/utils/class_utils.py

```diff
@@ -0,0 +1,6 @@
+import importlib
+
+
+def _get_class_from_string(fully_qualified_class_name):
+    module, class_name = fully_qualified_class_name.rsplit(".", maxsplit=1)
+    return getattr(importlib.import_module(module), class_name)
```

Comment:
add a unit test for this, otherwise LGTM

Reply:
Added a test.